queries/geography.py

```diff
@@ -173,6 +173,7 @@ def QGeoLocationDescQuery(node, params, result):
     (r"norður kóre", "Norður-Kóre"),
     (r"norður kaledón", "Norður-Kaledón"),
     (r"^seychelles.+$", "Seychelles"),
+    (r"^taiwans$", "Taiwan"),
```

Comment:
What about 'taíwan' or 'taívan' - does the speech recognizer not return those forms?

Reply:
"taívan" and "tævan" already handled (both in BÍN) but the speech recognizer sometimes returns "taiwan".